core/vm/opcodes.go

```diff
@@ -530,6 +537,7 @@ var stringToOp = map[string]OpCode{
 	"LOG2":           LOG2,
 	"LOG3":           LOG3,
 	"LOG4":           LOG4,
+	"PAYGAS":		  PAYGAS,
```

Comment:
Looks like there might be an extra tab here.

Reply:
I hate tabs, and I hate horizontal alignment.

Reply:
I hate tabs, and I hate horizontal alignment.